Feature/komarch 77 issuu

igor-kamil opened the pull request:
admin:
<img width="1134" alt="Screenshot 2021-05-04 at 23 15 34" src="https://user-images.githubusercontent.com/2682941/117071061-df37a980-ad2e-11eb-8ff1-0ea9bd8a1c64.png">

home page:
![Opera Snapshot_2021-05-04_231613_new komarch local](https://user-images.githubusercontent.com/2682941/117071097-e8c11180-ad2e-11eb-9320-822473018754.png)
<img width="323" alt="Screenshot 2021-05-04 at 23 17 04" src="https://user-images.githubusercontent.com/2682941/117071107-eb236b80-ad2e-11eb-9ba6-804723c73013.png">

ajobi commented:
I have encountered this issue with file manager while testing:
![Screenshot 2021-05-05 at 08 09 54](https://user-images.githubusercontent.com/22252756/117102932-9a853000-ad79-11eb-837a-86b9124e7ca7.png)

I have updated the composer dependencies and run the database migration. Have I missed something here?

igor-kamil commented:
> I have encountered this issue with file manager while testing:
> ![Screenshot 2021-05-05 at 08 09 54](https://user-images.githubusercontent.com/22252756/117102932-9a853000-ad79-11eb-837a-86b9124e7ca7.png)
> 
> I have updated the composer dependencies and run the database migration. Have I missed something here?

thank you for reporting. I updated setup instructions the README. does `php artisan storage:link` fixed the issue for you?

ajobi reviewed (approved):
Your suggestion solved the problem. Works properly in my eyes 👍
